ml-agents/setup.py

```diff
@@ -25,9 +25,11 @@
         'Programming Language :: Python :: 3.6'
     ],
 
-    packages=find_packages(exclude=['tests', 'tests.*', '*.tests', '*.tests.*']),  # Required
+    packages=['mlagents.trainers'],  # Required
+    zip_safe=False,
 
     install_requires=[
+        'mlagents_envs==0.7.0',
```

Comment:
We need to add to the checklist that this needs to be ticked at every release.

Reply:
Done()